pkg/restapi/issuer/operation/operations.go

```diff
@@ -519,12 +520,29 @@ func (o *Operation) createIssuerProfile(pr *ProfileRequest) (*vcprofile.DataProf
 
 	created := time.Now().UTC()
 
+	edvVaultID, err := o.createIssuerProfileVault(pr.Name)
+	if err != nil {
+		return nil, err
```

Comment:
I suggest wrapping this error - it will make debugging any issues later easier

Reply:
fixed!